client/main.less

```diff
@@ -1,5 +1,5 @@
-@phone: ~"(max-width: 767px)";
-@tablet: ~"(min-width: 767px) and (max-width: 991px)";
+@phone: ~"(max-width: 360px)";
+@tablet: ~"(min-width: 361px) and (max-width: 991px)";
```

Comment:
Nie za mało? Telefony są większe przecież. Jak już to można dać jeszcze small-phone